app/src/main/java/com/example/githubclone/presentation/loginscreen/LoginScreen.kt

```diff
@@ -0,0 +1,155 @@
+package com.example.githubclone.presentation.loginscreen
+
+import android.app.Activity
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.background
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.githubclone.R
+import com.example.githubclone.presentation.common.signIn
+import com.example.githubclone.ui.theme.EnterpriseButtonColor
+import com.example.githubclone.ui.theme.SignInButtonColor
+
+@Composable
+fun LoginScreen(
+    navigateToHomeScreen: () -> Unit,
+) {
+    val activity = LocalContext.current as Activity
+    val showLoader = rememberSaveable { mutableStateOf(false) }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(if (isSystemInDarkTheme()) Color(0xFF050505) else Color.White),
```

Comment:
Should this color come from theme ?

Reply:
Yes, I have fixed this and other colors as well in LoginScreen